js/components/portal-dashboard/student-answers.tsx

```diff
@@ -113,4 +141,5 @@ export class StudentAnswers extends React.PureComponent<IProps> {
       <div className={`${css.progressIcon} ${cssClass}`}/>
     );
   }
+
```

Comment:
extra space? line 144

Reply:
That's just the spacing format that I prefer.  I tend to put an empty line before/after/between functions for readability.